### PR TITLE
[#394] feat: update link of pycontw blogger

### PR DESCRIPTION
**Pull request by yungshenglu**
## Types of changes

* **Bugfix**
* **Refactoring**

## Description

- 更新 Blogger 網址為：https://conf.python.tw
- Refactor `BulletinCardCollection` 和 `BulletinCard`
    - 將原本 i18n 裡面非 i18n 相關的資料抽離到 `configs/pageLanding.js`
- Refactor `FooterSocial` 和 `FooterHistory`
    - 將資料抽離到 `configs/pageLanding.js`

## Related Issue
#394 

## Additional context
NA

### CODE REVIEW

**Comment by josix** [comment]
@SivanYeh @erik1110 please take a look about the refactroing (extracting data layer, new css coding style). And feel free to leave any thoughts, thanks!

**Review by tim08094495757 — COMMENTED** [review]
Thanks, @yungshenglu and @Josix.
These are feedbacks:
1. In BulletinCard.vue, classes like bulletin--card__content are removed, but classes like bulletin--card__header are still survived~ => Need a explanation. 
2. Youtube.svg to YouTube.svg => Now YouTube.svg is missing.

**Comment by yungshenglu** [comment]
> Thanks, @yungshenglu and @josix. These are feedbacks:
> 
> 1. In BulletinCard.vue, classes like bulletin--card__content were removed, but classes like bulletin--card__header are still survived~ => Need a explanation.
> 2. Youtube.svg to YouTube.svg => Now YouTube.svg is missing.

1. The reason why I don't remove `bulletin--card__header` is that there are have `media-query` in it which need extra check  between the design guideline and the support in Tailwind

File: configs/pageLanding.js
Comment (josix) [inline configs/pageLanding.js:1]: 😍😍😍

File: components/core/footer/FooterSocial.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.core-footerIcon:hover {
+    filter: brightness(0) invert(1) url(#hover-color);
+}
+
+@media (min-width: 768px) {
+    .core-footerIcon {
+        width: 33px;
+        height: 33px;
+    }
+}
```
Comment (josix) [inline components/core/footer/FooterSocial.vue:59]: **question:** I found that we have defined the `md` breakpoint prefix as `@media (min-width: 768px) {....}` in [`tailwind.config.js`](https://github.com/yungshenglu/pycontw-frontend/blob/david/394/tailwind.config.js#L99). I guess we could just adopt utility class `md:w-[33px] md:h-[33px]`for the class `core-footerIcon`?

File: components/bulletins/BulletinCard.vue [hunk 2]
```diff
@@ -72,16 +87,6 @@ export default {
...
-}
-
-.bulletin--card__content {
-    @apply absolute;
-    top: 30%;
-}
-
 .bulletin--card__header {
     @apply font-serif text-base text-center mt-6 mb-5 font-bold;
     @media (min-width: 375px) {
```
Comment (josix) [inline components/bulletins/BulletinCard.vue:92]: **question:** how about using `xs:text-xl xs:mt-8 xs:mb-5`

File: components/core/footer/FooterSocial.vue [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<template>
+    <div class="w-full flex justify-center items-center flex-wrap">
+        <ext-link
+            v-for="(item, index) in dataConfig"
+            :key="`landing-footer-icon-${index}`"
```
Comment (josix) [inline components/core/footer/FooterSocial.vue:5]: **suggestion:** I guess we also need to define the convention of naming the keys of items in array. e.g. `[page]-[block]-[element]-[index]` or something like that.

**Review by josix — CHANGES_REQUESTED** [review]
Thanks for the hard work @yungshenglu. The whole refactoring about static data control looks really neat. Thanks!

**Review by baby230211 — APPROVED** [review]

File: components/core/footer/FooterSocial.vue (hunk 1 above)
Comment (yungshenglu) [reply components/core/footer/FooterSocial.vue:59]: If we have already defined, then I can replace them.

File: components/core/footer/FooterSocial.vue (hunk 3 above)
Comment (yungshenglu) [reply components/core/footer/FooterSocial.vue:5]: This is the part I missing in last meeting. There are so many kinds of naming convention in the repository. I totally agree that we should define one of our naming convention here.

File: components/core/footer/FooterSocial.vue (hunk 1 above)
Comment (josix) [reply components/core/footer/FooterSocial.vue:59]: yeah, here it is https://github.com/yungshenglu/pycontw-frontend/blob/david/394/tailwind.config.js#L99

File: components/bulletins/BulletinCard.vue (hunk 2 above)
Comment (yungshenglu) [reply components/bulletins/BulletinCard.vue]: Fixed.

**Review by josix — APPROVED** [review]
LGTM!

**Pull request merged**